Bash_scripts/HW20/roles/hw201/tasks/docker_start.yml

```diff
@@ -0,0 +1,16 @@
+#Install Docker engine and Start docker service
+
+- name: "Install Docker Engine"
```

Comment:
This should be under docker.yml as it's not related to start action

Reply:
Made hendler to start, returned the rest of the installation to docker.yml